rust/Cargo.toml

```diff
@@ -38,3 +39,8 @@ members = [
 
 [profile.bench]
 lto = true
+
+# Avoid libllvm/libclang dep
```

Comment:
What's the rationale here?

Reply:
This is currently a PR but upstream seems kind of dead :/

Without this we'll pull in libclang and libllvm (both pretty heavy deps) and also pull in clap (which bloats binary size by ~1.5MB). I hope this will be merged soon.

Reply:
Can we include a link to the PR in the code, with a note to remove this when it's merged? This will greatly benefit future archaeologists. 🕵️‍♀️

Reply:
Right, sounds fair

Reply:
Done